File: Prak 7/C/PRAK601-2210817110005-MIZZUDDINABDISSALAM.c
#include <stdio.h>

int main(){
    int brs, klm, i, j, in;
    scanf("%d %d", &brs, &klm);
    int matrik[brs][klm];
    for (i = 0; i < brs; i++){
        for (j = 0; j < klm; j++){
        scanf("%d", &in);
        matrik[i][j] = in;}
    }
    printf("\n");
    for (i = 0; i < brs; i++){
        for (j = 0; j < klm; j++){
            printf("%d ", matrik[i][j]);}
        printf("\n");}
return 0;}

File: Prak 7/Python/PRAK601-2210817110005-MIZZUDDINABDISSALAM.py
klm, brs = map(int, input().split())
angka = list(map(int, input().split()))
matrik = []
a = 0
for i in range(klm) :
    matrik.append(angka[a : a + brs])
    a = a + brs
for klm in matrik :
    for angka in klm :
        print(angka, end=' ')
    print()
